File: LeetCode-Problems/Easy/Max-Nesting-Depth.py
# Python
class Solution(object):
    def maxDepth(self, s):
        """
        :type s: str
        :rtype: int
        """
        maxCount = 0
        count = 0
        for x in range(0, len(s)):
            if s[x:x+1] == "(":
                count = count + 1
            if s[x:x+1] == ")":
                count = count - 1
            if count > maxCount:
                maxCount = count
        return maxCount
        

File: LeetCode-Problems/Easy/Max-Nesting-Depth.java
//Java
class Solution {
    public int maxDepth(String s) {
        int maxCount = 0;
        int count = 0;
        for (int i = 0; i < s.length(); i++){
            if (s.charAt(i) == '('){
                count++;
            }
            if (s.charAt(i) == ')'){
                count--;    
            }
            if (count > maxCount){
                maxCount = count;
            }
        }
        return maxCount;
    }
}
